go/vt/sqlparser/parse_test.go

```diff
@@ -44,13 +47,13 @@ func TestValid(t *testing.T) {
 		input:  "select - -1 from t",
 		output: "select 1 from t",
 	}, {
-		input:  "select 1 from t // aa",
+		input:  "select 1 from t // aa\n",
```

Comment:
I added \n as a hack. Below the ParseNext concats all the input strings together, to make one really long test input. However, a comment that doesn't end with a newline basically wipes out the full string.